src/celeritas/optical/CerenkovParams.cc

```diff
@@ -58,10 +56,9 @@ CerenkovParams::CerenkovParams(SPConstProperties properties)
                                 * (1 / ipow<2>(refractive_index[i - 1])
                                    + 1 / ipow<2>(refractive_index[i]));
         }
-        angle_integral.push_back(
-            build_angle_integral(make_span(energy), make_span(integral)));
+
+        build_angle_integral(make_span(energy), make_span(integral));
     }
-    CELER_ASSERT(angle_integral.size() == host_ref.refractive_index.size());
```

Comment:
I think this should be restored, and you can use `data.angle_integral` instead of the now-deleted builder